libhsakmt/src/fmm.c

```diff
@@ -1916,7 +1916,7 @@ static void *fmm_allocate_host_gpu(uint32_t gpu_id, uint32_t node_id, void *addr
 		 * fork. This avoids MMU notifiers and evictions due to user
```

Comment:
If we do proceed with this change, these comments should also be removed as they were based on DONTFORK flag.
But I think the DONTFORK flag is needed for CRIU.